workspaces/sandbox/plugins/sandbox/app-config.dynamic.yaml

```diff
@@ -1,19 +1,91 @@
-dynamicPlugins:
-  frontend:
-    red-hat-developer-hub.backstage-plugin-sandbox:
-      appIcons:
-        - name: homeIcon
-          importName: SandboxHomeIcon
-        - name: activitiesIcon
-          importName: SandboxActivitiesIcon
-      dynamicRoutes:
-        - path: /home
-          importName: SandboxPage
-          menuItem:
-            icon: homeIcon
-            text: Home
-        - path: /activities
-          importName: SandboxActivitiesPage
-          menuItem:
-            icon: activitiesIcon
-            text: Activities
+includes:
+  - dynamic-plugins.default.yaml
+plugins:
+  - disabled: false
+    package: oci://quay.io/<CONTAINER_IMAGE>!red-hat-developer-hub-backstage-plugin-sandbox
```

Comment:
just for my understanding - where is `<CONTAINER_IMAGE>` coming from ? 

isn't this already defined in the sandbox-sre configuration ?

Reply:
This is a reference file for future use. In case the cluster configmaps are lost.